azure_monitor/README.rst

```diff
@@ -0,0 +1,17 @@
+Azure Monitor Exporters using OpenTelemetry
+===========================================
+
+This library provides integration with Azure Monitor using OpenTelemetry.
+
+Installation
+------------
+
+::
+
+    pip install azure-monitor-opentelemetry-exporters-python
+
+References
+----------
+
+* `Azure Monitor <https://docs.microsoft.com/azure/azure-monitor/>`_
+* `OpenTelemetry Project <https://opentelemetry.io/>`_
```

Comment:
https://github.com/open-telemetry/opentelemetry-python would be better here, saving some clicks and confusion for people using this